int -> uint64_t

ahmedcharles opened the pull request:
Using int results in a `sext.w` instruction, which is confusing when
looking at the disassembly.

optimisticside commented:
`uint64_t` isn't defined since `stdint.h` is not included. Instead, `uint64` is used in its place.

ahmedcharles committed:
int -> uint64_t

Using int results in a `sext.w` instruction, which is confusing when
looking at the disassembly.

echtwerner commented:
close this PR, please

kaashoek commented:
Closing expired pull request.
